db/migrate/20170605204001_create_chembl_molecules.rb

```diff
@@ -0,0 +1,40 @@
+class CreateChemblMolecules < ActiveRecord::Migration
+  def change
+    create_table :chembl_molecules do |t|
+      t.integer :molregno
+      t.string :pref_name, :limit => 255
```

Comment:
Its totally your call (and I wouldn't probably bother to go back and change this at this point), but I generally would lean towards using `text` rather than `string` (which will end up as `varchar`under the hood) in the future. There's no performance disadvantage to using `text` in posgres and it doesn't require you to specify a limit in advance (see the "tip" box here: https://www.postgresql.org/docs/current/static/datatype-character.html).